### PR TITLE
A few changes.

### PR DESCRIPTION
**Pull request by gabrielvicenteYT**
EvergreenHUD on top.

### CODE REVIEW

File: docs/1.8.9/lunar_client.md [hunk 1]
```diff
@@ -99,7 +99,7 @@
 * Block Outline - [Aycy's BlockOverlay](https://hypixel.net/threads/forge-1-8-9-block-overlay-v4-0-3.1417995/)
 * Item Physics - [CreativeMD's ItemPhysic Lite](https://www.curseforge.com/minecraft/mc-mods/itemphysic-lite/files/all?filter-game-version=2020709689%3A5806)
 * TNT Timer Mod - [Sk1er's TNT Time](https://sk1er.club/mods/tnttime)
-* Hitbox - [Polyfrost's REDACTION](https://github.com/Polyfrost/REDACTION/releases/latest)
+* Hitboxes - [Polyfrost's REDACTION](https://github.com/Polyfrost/REDACTION/releases/latest) / [Sk1er's Old Animations](https://sk1er.club/beta) (adds back 1.7 style hitboxes) **_and_** [Polyfrost's OverflowAnimations](https://github.com/Polyfrost/OverflowAnimations/releases/latest) {yes, you need **_both_** Sk1er's OA and Polyfrost's OA.}
```
Comment (Mqisty) [inline docs/1.8.9/lunar_client.md:102]: ?

Comment (MicrocontrollersDev) [reply docs/1.8.9/lunar_client.md:102]: yeah this a bit weird. i’m wondering if i should hold this pr off until oneconfig releases so i can put in the actual good hitboxes mod. according to wyvest oneconfig will release yesterday so we’ll see

Comment (MicrocontrollersDev) [reply docs/1.8.9/lunar_client.md:102]: yeah this on hold since it’ll probably  be changed soon anyways, will make necessary edits when that time comes

**Comment by MicrocontrollersDev** [comment]
homie unironically used a label 💀

**Comment by Mqisty** [comment]
> homie unironically used a label 💀

image

**Comment by MicrocontrollersDev** [comment]
> > homie unironically used a label 💀
> 
> image

ain’t me 🤷

**Comment by MicrocontrollersDev** [comment]
not much of a fan of the changes here and the hitboxes one will wait for nox's. That's the only change here I like so I'll just close this

**Comment by Mqisty** [comment]
my bad

**Pull request closed without merging**